test/Microsoft.Azure.ServiceBus.UnitTests/QueueSessionTests.cs

```diff
@@ -188,6 +189,28 @@ public sealed class QueueSessionTests
             }
         }
 
+        [Theory]
+        [MemberData(nameof(TestPermutations))]
+        [DisplayTestMethodName]
+        async Task ServerWaitTimeoutTest(string queueName, int messageCount = 1)
```

Comment:
@nemakam sir, thank you from the bottom of my ❤️ for the proper test name. I can die a happy man now. @jtaubensee watch and learn 😛